### DISCUSSION TITLE
Install command not working on mac

### DISCUSSION THREAD

**Discussion by mveer08** [Homebrew/discussions#4554] [Getting started]
### Output of `brew config`

```shell
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Above command is not working, it is throwing error timeout in terminal.

If I browse link https://raw.githubusercontent.com/Homebrew/install/HEAD/install then it throws error 404
```

### Output of `brew doctor`

```shell
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Above command is not working, it is throwing error timeout in terminal.

If I browse link https://raw.githubusercontent.com/Homebrew/install/HEAD/install then it throws error 404
```

### Description of issue

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Above command is not working, it is throwing error timeout in terminal.

If I browse link https://raw.githubusercontent.com/Homebrew/install/HEAD/install then it throws error 404

**Comment by gromgit** [comment]
Please post the _exact_ output of the command you ran. Saying "error timeout in terminal" tells us nothing about what installer stage encountered the error, nor what the details are.

This is what's meant by the tech support catchphrase **show, don't tell**: ***Show*** us exactly what you did and what you saw, don't _**tell**_ us a story with huge gaps in it.

Also, please post the output _as text_, **NOT** screenshots. Pictures are really hard to extract textual details from on a mobile phone, for instance.

> If I browse link https://raw.githubusercontent.com/Homebrew/install/HEAD/install then it throws error 404

Yes, that's expected, because that's not the same URL as in the installer command. Please look carefully.
